### ISSUE TITLE
Instructors are confused by {The student's unique submission url appears here}

### ISSUE THREAD

**Issue by damithc** [TEAMMATES/teammates#5314]

**Comment by YongJieYongJie** [comment]
Maybe we can add a separator:
`==== Email Message Body as Seen by Students ====`

like so:
![image](https://cloud.githubusercontent.com/assets/14101781/15350164/4331ce44-1d0a-11e6-9bcf-3fafcf9e7664.png)


for reference, right now the email looks something like this:
![image](https://cloud.githubusercontent.com/assets/14101781/15349735/a77f8fb0-1d07-11e6-8ea7-776db4f07ef4.png)

**Comment by damithc** [comment]
The separator helps, but not enough. The instructor will still think the missing link is a bug.

**Comment by YongJieYongJie** [comment]
Maybe rephrase
`{The student's unique submission url appears here}`
to something that reads more like a prose:
`{in the actual emails sent to the students, this will be the unique link}`

something like this:
![image](https://cloud.githubusercontent.com/assets/14101781/15352597/89ab58a6-1d17-11e6-8447-ca45ae1f1f9b.png)

**Comment by rclakmal** [comment]
Wouldn't the message be more simpler if it looked like 
**To submit feedback for the above session, please click [here](https://teammatesv4.appspot.com/).**

**Comment by damithc** [comment]
Yes, I think @rclakmal 's idea is a good one.
